### PR TITLE
[ignore] documentation updates to mention cloud network controller in…

### LINKED ISSUE #1034: Update documentation for cloud resources and data sources to reflect Cloud Network Controller
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The cloud section with resources and data sources within the ACI provider documentation is reflecting ACI Cloud and Cloud APIC, which has been rebranded to Cloud Network Controller. Although the provider continues to be the same, the ask is to update the documentation to reflect the new name for consistency with other public documentation. 

Thank you!

### PR DESCRIPTION
**Pull request by akinross**
…stead of aci cloud, cloud apic or cloud aci

fixes #1034

### CODE REVIEW

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Review by lhercot — CHANGES_REQUESTED** [review]
Please apply same to rest of changes. If the string was ACI Cloud -> Cloud Network Controller Cloud.

File: website/docs/d/cloud_account.html.markdown [hunk 1]
```diff
@@ -4,13 +4,13 @@ layout: "aci"
 page_title: "ACI: aci_cloud_account"
 sidebar_current: "docs-aci-data-source-cloud-account"
 description: |-
-  Data source for ACI Cloud Account
+  Data source for Cloud Network Controller Cloud Account
```
Comment (shrsr) [inline website/docs/d/cloud_account.html.markdown:7]: Cloud Network Controller Cloud Account sounds confusing
Just a suggestion ->
Data source for the Cloud Account

**Review by shrsr — CHANGES_REQUESTED** [review]

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Review by sajagana — APPROVED** [review]
LGTM!

**Review by shrsr — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**